db/migrations/20240620092323_goteo_form_honeypot.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * Migration Task class.
+ */
+class GoteoFormHoneypot
+{
+    public function preUp()
+    {
+        // add the pre-migration code here
+    }
+
+    public function postUp()
+    {
+        // add the post-migration code here
+    }
+
+    public function preDown()
+    {
+        // add the pre-migration code here
+    }
+
+    public function postDown()
+    {
+        // add the post-migration code here
+    }
+
+    /**
+     * Return the SQL statements for the Up migration
+     *
+     * @return string The SQL string to execute for the Up migration.
+     */
+    public function getUpSQL()
+    {
+        return "
+            CREATE TABLE `form_honeypot` (
+                `id` BIGINT(20) UNSIGNED NOT NULL AUTO_INCREMENT,
+                `trap` VARCHAR(50) NOT NULL,
+                `prey` TEXT,
+                `template` TEXT,    
```

Comment:
Should we create a datetime field? So that we can get info about when we have a spike on spam?